[PHPMD] The document for Sider's recommended set of rules. (#637)
sider/sider-docs: docs/tools/php/phpmd.md

To start using PHPMD, enable it in your [repository settings](../../getting-started/repository-settings.md).

## Default Configuration for PHPMD

Sider provides our [recommended ruleset](https://github.com/sider/runners/blob/HEAD/images/phpmd/sider_recommended_phpmd.xml) for PHPMD.

> This configuration is used when you have no PHPMD configurations or `sider.yml` in your repository.

Reviewer: According to the source code, our recommended file is used when the `rule` option is not specified. How about making the same sentence as cppcheck.

```suggestion
This configuration is used when the `rule` option is not specified in `sider.yml`.
```

Reviewer: And also:
- In the options table, the default value of the `rule` should be `sider_recommended_phpmd.xml`
- The link "If omitted, Sider uses the default configuration." in the `rule` section should be `sider_recommended_phpmd.xml`

Author: Thanks, I fixed it.
